packages/fxa-auth-server/lib/routes/linked-accounts.ts

```diff
@@ -237,6 +240,9 @@ export class LinkedAccountHandler {
           locale: request.app.acceptLanguage,
         });
         await this.db.createLinkedAccount(accountRecord.uid, userid, provider);
+
+        await this.profile.updateProfileName(accountRecord.uid, idToken.name);
```

Comment:
I think we should set display name for both new users and users who are linking third party for the first time.

Reply:
The `name` in id token might not always be present according to Google docs

> When name claims are present, you can use them to update your app's user records. Note that this claim is never guaranteed to be present.